packages/klighd-core/src/options/option-models.ts

```diff
@@ -23,6 +23,12 @@ export interface RenderOption {
     initialValue: any;
     currentValue: any;
     description?: string;
+    /** The category this RenderOption is part of. */
+    renderCategory?: string;
+    /** The values this RenderOption has, if it's type is {@link TransformationOptionType.CHOICE}. */
+    renderChoiceValues?: any[];
```

Comment:
These are exactly the same as previously defined in the `SynthesisOption` interface a few lines below under the field names `category` and `values` (matching their Java counterparts). I suggest to rename these fields to `category` and `values` and remove the fields from the `SynthesisOption` interface for consistency between the option types and to remove duplicate fields on synthesis options.

Reply:
But the fields are not the same a renderCategory is only a string since the render option registration process does not allow to add a singleton category option on the client side.

Reply:
Maybe RenderOption is just not a good super type. I suggest to make a general option type and extend it by renderCategory and category for RenderOptions and SynthesisOptions

Reply:
I see, the category serialized from the server side is also not serialized into a singleton, currently causing multiple equal categories in the option data, of which only the string ID will be used anyway. I guess leaving it like this is okay, as 'correctly' using an option as the category with just `{id: theId} as SynthesisOption` could potentially lead to other problems.

But the `choiceValues`/`values` fields should be and do the same thing I think

Reply:
Yes, I did it that way.